context.go

```diff
@@ -43,6 +43,9 @@ const BodyBytesKey = "_gin-gonic/gin/bodybyteskey"
 // abortIndex represents a typical value used in abort functions.
 const abortIndex int8 = math.MaxInt8 >> 1
 
+// &CtxKey is the key that a Context returns itself for.
+var CtxKey int
```

Comment:
Why not use a string value like in `BodyBytesKey`?
```suggestion
const ContextKey = "_gin-gonic/gin/contextkey"
```

Reply:
There are similar implementations I've seen in Go inbuilt library code.

Let me change it as per your suggestion.